### ISSUE TITLE
New rule: Non-flags enums should not be marked with "FlagsAttribute"

### ISSUE THREAD

**Issue by meziantou** [meziantou/Meziantou.Analyzer#25]
```csharp
    [FlagsAttribute] // Noncompliant, Orange is neither a power of two, nor a combination of any of the defined values
    public enum Color
    {
        None    = 0,
        Red     = 1,
        Orange  = 3, 
        Yellow  = 4,
    }
```
